### DISCUSSION TITLE
Help in creation of Graphics software program

### DISCUSSION THREAD

**Discussion by Artan-Botianni** [community/community#52278] [Programming Help] [answered]
### Body

Hello.
I'm making Graphics software program like Photoshop and Blender, that is, a 2D and 3D Graphics editor & I need a proficient Graphics software programmer to help. Need Q&A or more.
Needed knowledge and experience: 3D computer graphics software, 2D and 3D Graphics editor, Raster graphics editor, Computer-Aided Design (plans), Image Recognition, Texture mapping, Image manipulation software.
Using programming languages by me: C++ and Python. Most of the Free and open-source software needed for the project is written in them. My OS is Windows.
Can you help or may be your know other matching person ? If you matching - write.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by edwinm** [comment]
You can't post jobs here.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Artan-Botianni , we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
